docs/snippets/src/main.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+#! [import]
+import openvino.runtime as ov
+#! [import]
+
+#! [part1]
+core = ov.Core()
+#  model is ov.Model
+#  compiled_model is ov.CompiledModel
```

Comment:
why is it here?

Reply:
I added these comments to align examples with C++ here I show objects which can be used below